src/DynamoCoreWpf/UI/GuidedTour/GuideFlowEvents.cs

```diff
@@ -50,6 +51,7 @@ internal static void OnGuidedTourFinish(string name)
             {
                 isAnyGuideActive = false;
                 GuidedTourFinish(new GuidedTourStateEventArgs(name));
+                Logging.Analytics.TrackEvent(Logging.Actions.Completed, Logging.Categories.GuidedTourOperations, Resources.ResourceManager.GetString(name).Replace("_", ""));
```

Comment:
@zeusongit I believe the name here is already the tour name so getting the resource will return `null`

Reply:
Also when I exit the tour, I see `end` and `completed` analytics event both triggered